how-to/workspace-platform-starter/public/common/views/manager-portal/common/common.js

```diff
@@ -26,23 +24,33 @@ export async function loadData(filename) {
 }
 
 /**
- * Load the team data.
+ * Use the user data to create a team list.
+ * @param users The user data.
  * @returns The team data.
  */
-export async function loadTeamData() {
-	const teamData = await loadData('team.json');
-	const updatedTeamMembers = [];
+export async function addUserDates(users) {
 	// For each teamMember, randomize some dates for them.
-	for (const member of teamData) {
-		const updated = member;
-		const approvedDays = member.leave.approved;
-		const awaitingDays = member.leave.awaitingApproval;
-		updated.leave.approved = modifyDates(approvedDays);
-		updated.leave.awaitingApproval = modifyDates(awaitingDays);
-		updated.leave.lastUpdated = new Date();
-		updatedTeamMembers.push(updated);
+	for (const user of users) {
+		user.leave = {
+			alloted: 20,
+			approved: [
+				'2022-10-01',
+				'2022-10-02',
+				'2022-11-09',
+				'2022-11-10',
+				'2022-11-11',
+				'2022-12-28',
+				'2022-12-29'
+			],
+			awaitingApproval: ['2023-01-01'],
```

Comment:
should this be hardcoded...I thought that Joe had added dynamic elements so that we wouldn't have stale dates?

Reply:
They are the basis on which Joes code generates the dynamic dates